sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/validation/CreateTableValidator.kt

```diff
@@ -25,7 +25,7 @@ import com.squareup.sqldelight.types.ForeignKey
 
 internal class CreateTableValidator(var resolver: Resolver) {
   fun validate(createTable: SqliteParser.Create_table_stmtContext) {
-    val resolution = listOf(resolver.resolve(createTable)).filterNotNull()
```

Comment:
this one seems meaningful maybe

Reply:
The IDE reports them as useless which I took to mean a collection of non-nullable types was being filtered for nulls. I'll double check.

Reply:
`resolver.resolve` return `Table` which becomes `List<Table>` in which there cannot be a null.

Reply:
dope dope thanks for checkiiin